src/vs/editor/common/model/editableTextModel.ts

```diff
@@ -521,6 +519,8 @@ export class EditableTextModel extends TextModelWithDecorations implements Edito
 					this._lines[startLineNumber + insertingLinesCnt - 1].append(deferredEventsBuilder.changedMarkers, leftoverLine);
 
 					this.emitModelContentChangedLinesInsertedEvent(startLineNumber + editingLinesCnt + 1, startLineNumber + insertingLinesCnt, newLinesContent.join('\n'));
+				} else {
+					this.emitModelContentChangedLinesInsertedEvent(startLineNumber + editingLinesCnt + 1, startLineNumber + insertingLinesCnt, '\n');
```

Comment:
This line fixes the "crash".  An event must be emitted, it doesn't have to be this one and I'm not sure if all the parameters are perfect.